Xamarin.Forms.Platform.iOS/Renderers/EntryRenderer.cs

```diff
@@ -192,5 +220,63 @@ void UpdateText()
 			if (Control.Text != Element.Text)
 				Control.Text = Element.Text;
 		}
+
+		void UpdateDisabledSelectorActions()
+		{
+			(_uiTextField as UITextFieldWrapper).DisabledSelectorActions = Element.On<PlatformConfiguration.iOS>().DisabledSelectorActions();
+		}
+	}
+
+	class UITextFieldWrapper : UITextField
+	{
```

Comment:
`CanPerform` does not work unless `UITextField` is subclassed; hence the wrapper.